include/sdf/Root.hh

```diff
@@ -58,13 +58,13 @@ namespace sdf
     /// \brief Default constructor
     public: Root();
 
-    /// \brief Get the name of the world
+    /// \brief Get the name of the world without loading the entire world
```

Comment:
Can you add that users shouldn't normally need to use this API. Also please mention that since this doesn't load the world, it might return the world name even if the file is not a valid SDFormat file.

Reply:
https://github.com/gazebosim/sdformat/pull/1027/commits/a7a68576a6380d4bb670de5cb8fc79851e37b713